### PR TITLE
Adds rule reminders to revivals of various kinds.

### PR DESCRIPTION
**Pull request by Absolutely-Steph**
## About The Pull Request
Just gives the revived person some big red text making it very clear what they can remember or not.

## Why It's Good For The Game
People jumping out of the cloner and yelling GEORGE IS A LING is not nice, Now I didn't say that actually happens often enough to warrant this, but it's always good to make sure the players know the rules.

## Changelog
:cl:
add: Adds rule reminders to revivals of various kinds.
/:cl:

### CODE REVIEW

File: code/game/machinery/cloning.dm [hunk 1]
```diff
@@ -280,6 +280,7 @@
 		H.set_cloned_appearance()
 
 		H.set_suicide(FALSE)
+	to_chat(H, "<span class='warning'><b>You are being cloned. As per our rules, you cannot remember how you died.</b></span>") //NSV13
```
Comment (covertcorvid) [inline code/game/machinery/cloning.dm:283]:
```suggestion
	to_chat(H, "<span class='warning'><b>You are being cloned. You cannot remember how you died.</b></span>") //NSV13
```

File: code/game/objects/items/defib.dm [hunk 2]
```diff
@@ -654,6 +654,7 @@
 					H.revive()
 					H.emote("gasp")
 					H.Jitter(100)
+					to_chat(H, "<span class='warning'><b>You have been revived via defibrillation. As per our rules, you cannot remember how exactly you died. But you may remember certain details such as the weapon used or the place of death.</b></span>") //NSV13
```
Comment (covertcorvid) [inline code/game/objects/items/defib.dm:657]:
```suggestion
					to_chat(H, "<span class='warning'><b>You have been revived via defibrillation. You cannot remember how exactly you died (or who killed you), but you may remember certain details such as the weapon used or the place of death.</b></span>") //NSV13
```

File: code/modules/surgery/advanced/revival.dm [hunk 3]
```diff
@@ -68,6 +68,7 @@
 	if(target.revive())
 		target.visible_message("...[target] wakes up, alive and aware!")
 		target.emote("gasp")
+		to_chat(target, "<span class='warning'><b>You have been revived via Revival Surgery. As per our rules, you cannot remember how exactly you died. But you may remember certain details such as the weapon used or the place of death.</b></span>") //NSV13
```
Comment (covertcorvid) [inline code/modules/surgery/advanced/revival.dm:71]:
```suggestion
		to_chat(target, "<span class='warning'><b>You have been revived via Revival Surgery. You cannot remember how exactly you died (or who killed you), but you may remember certain details such as the weapon used or the place of death.</b></span>") //NSV13
```

**Review by covertcorvid — APPROVED** [review]
Good change. I suggested removing references to the rules for immersion and making it more explicit that we don't want validhunting.

File: code/game/machinery/cloning.dm (hunk 1 above)
Comment (Absolutely-Steph) [reply code/game/machinery/cloning.dm:283]: I thought adding the rules would make it very fucking clear that this is punishable, but if you think we should keep it IC then that's fine.

**Comment by Bokkiewokkie** [comment]
@Absolutely-Steph please do apply the changes Corvid requested

**Review by covertcorvid — APPROVED** [review]

**Pull request merged**